pkg/controller/common/controller_context.go

```diff
@@ -111,14 +111,15 @@ func CreateControllerContext(ctx context.Context, cb *clients.Builder) *Controll
 		klog.Warningf("unable to get owner reference (falling back to namespace): %v", err)
 	}
 
-	recorder := events.NewKubeRecorder(kubeClient.CoreV1().Events(MCONamespace), "cloud-controller-manager-operator", controllerRef)
```

Comment:
comment (non-blocking): Wow! I have no idea how / why this was set to `cloud-controller-manager-operator`.